kaolin/graphics/SoftRenderer.py

```diff
@@ -40,9 +40,11 @@
 import torch
 
 import kaolin as kal
-from .DifferentiableRenderer import DifferentiableRenderer
-from .Lighting import compute_ambient_light
-from .Lighting import compute_directional_light
+from DifferentiableRenderer import DifferentiableRenderer
```

Comment:
please use `.DifferentiableRenderer` relative import otherwise it will leads to import errors. Same for imports below.